src/inlineeditor.js

```diff
@@ -79,7 +80,8 @@ export default class InlineEditor extends Editor {
 	 * @inheritDoc
 	 */
 	get element() {
-		return this.ui.view.editable.element;
+		log.warn( 'deprecated-editor-element: The editor#element is deprecated.' );
+		return this.ui.element;
 	}
```

Comment:
Not sure if we need to keep this property. It has no docs and was not a part of the old API. It was just a mistake.

Reply:
It exists in the documentation (https://ckeditor.com/docs/ckeditor5/latest/api/module_editor-inline_inlineeditor-InlineEditor.html#member-element), because it is inherited/mixed from `EditorWithUI` so somebody may already have used it. That's the reason it's safer to deprecate than to remove.